compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -399,15 +389,13 @@ def start_endpoint(
         # place registration after everything else so that the endpoint will
         # only be registered if everything else has been set up successfully
         if not reg_info:
-            endpoint_uuid = Endpoint.get_or_create_endpoint_uuid(
-                endpoint_dir, endpoint_uuid
-            )
-            log.debug("Attempting registration; trying with eid: %s", endpoint_uuid)
+            endpoint_uuid = Endpoint.get_endpoint_id(endpoint_dir) or endpoint_uuid
+            log.debug("Attempting endpoint registration")
```

Comment:
Why remove the "trying with" datum?  If we have the information (even if `None`), why not share it?

Reply:
I removed it because the value could be `None`, but I'll put it back in.